kubernetes/known_labels_annotations.go

```diff
@@ -15,10 +15,10 @@ import (
 var knownLabelsAnnotations = map[string]string{
 	// core
 	api.LabelHostname:                            "",
+	api.LabelTopologyZone:                        "",
```

Comment:
These are actually the same as the the two I removed below – there is a comment in the k8s code to use these consts instead.